src/Microsoft.ML.Onnx/SaveOnnxCommand.cs

```diff
@@ -134,7 +136,50 @@ private void GetPipe(OnnxContextImpl ctx, IChannel ch, IDataView end, out IDataV
                 transform = (source = transform.Source) as IDataTransform;
             }
 
-            Host.AssertValue(source);
+            Contracts.AssertValue(source);
```

Comment:
>Contracts [](start = 12, length = 9)

Similar note here.

Reply:
Sure. Thank you.

---
In reply to: [245429058](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245429058) [](ancestors = 245429058)